<?xml version="1.0"?>
<launch>

  <arg name="source_file" default="lawn.mp4" />
    <node name="image_source_video" pkg="image_source" type="image_source_video.py">
        <param name="width" value="1280" />
        <param name="height" value="720" />
        <param name="frame_rate" value="15"/>
        <param name="source_file" value="$(arg source_file)"/>
    </node>

   <test test-name="test_ekf_localization_node_bag2_pose" pkg="robot_localization" type="test_ekf_localization_node_bag2" clear_params="true" time-limit="1000.0">
      <param name="final_x" value="1.0206"/>
      <param name="final_y" value="3.4292"/>
      <param name="final_z" value="0.7419"/>
      <param name="tolerance" value="0.1383"/>
      <param name="output_final_position" value="$(arg output_final_position)"/>
      <param name="output_location" value="$(arg output_location)"/>
    </test>

</launch>
